apps/react/src/features/studios/components/StudioSortSelect/StudioSortSelect.tsx

```diff
@@ -0,0 +1,75 @@
+import { useAppDispatch } from '@js-camp/react/store';
+import { resetCursor, setOrdering, setPaginationEvent } from '@js-camp/react/store/studio/slice';
+import { FormControl, IconButton, InputLabel, MenuItem, Select, SelectChangeEvent, Tooltip } from '@mui/material';
+import ArrowUpwardIcon from '@mui/icons-material/ArrowUpward';
+import ArrowDownwardIcon from '@mui/icons-material/ArrowDownward';
+import { FC, memo } from 'react';
+import { StudioSortDirection } from '@js-camp/core/models/enums/studio-sort-direction.enum';
+import { useSearchParams } from 'react-router-dom';
+
+import { sortValueFromQueryParams } from '../../utils/sortValueFromQueryParams';
+import { queryFromSortValue } from '../../utils/queryFromSortValue';
+
+import styles from './StudioSortSelect.module.css';
+
+const StudioSortSelectComponent: FC = () => {
+	const dispatch = useAppDispatch();
+	const [searchParams] = useSearchParams();
+	const ordering = searchParams.get('ordering');
+	const { sortValue, sortDirection } =
+		ordering !== null ? sortValueFromQueryParams(ordering) : { sortValue: null, sortDirection: null };
```

Comment:
To avoid recalculations you can wrap this code in `useMemo` hook:
```suggestion
	const { sortValue, sortDirection } = useMemo(() => {
		const ordering = searchParams.get('ordering');
		return ordering !== null ? sortValueFromQueryParams(ordering) : { sortValue: null, sortDirection: null };
	}, [searchParams]);
```